### DISCUSSION TITLE
Unable to configure Firefly

### DISCUSSION THREAD

**Discussion by yourfriendlynoob** [firefly-iii/firefly-iii#7884] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Installed firefly with yunohost on a debian 11 instance from ionos.
all ports except port 25 configured
port 25 blocked

Whoops! An error occurred.

Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Expected response code "250/251/252" but got code "553", with message "553 5.7.1 <shashankhrs@shashankhrs.in>: Sender address rejected: not logged in".

This error occured in file /var/www/firefly-iii/app/Jobs/MailError.php on line 103 with code 0.
More information

Please collect more information in the storage/logs directory where you will find log files. If you're running Docker, use docker logs -f [container]. You can read more about collecting error information in [the FAQ](https://docs.firefly-iii.org/faq/other#how-do-i-enable-debug-mode).

### Debug information

Debug information generated at 2023-08-22 00:00:00 Etc/UTC for Firefly III version **6.0.5**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.5 |
| Firefly III API | 2.0.1 |
| Build | (unknown), (unknown) |
| DB version | 19 (exp. 19) |
| PHP | `8.2.8` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Etc/UTC |
| Browser TZ | Asia/Kolkata |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 52428800 (50 MB) |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start |  |
| Session end |  |
| Session first |  |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0 |

### Expected behaviour

Unable to create an account or login

**Comment by JC5** [comment]
Sorry, I don't know a lot about Yunohost myself. Does it supply email login data?
